aten/src/ATen/native/NonSymbolicBC.h

```diff
@@ -10,4 +10,14 @@ namespace native {
 // In those cases, we will duplicate the signature here with non-symbolic ints, and also duplicate the C++ implementation.
 TORCH_API at::Tensor reshape(const at::Tensor& self, at::IntArrayRef proposed_shape);
 TORCH_API at::Tensor narrow(const at::Tensor& self, int64_t dim, int64_t start, int64_t length);
+TORCH_API at::Tensor _sparse_coo_tensor_unsafe(const at::Tensor & indices, const at::Tensor & values, at::IntArrayRef size, c10::optional<at::ScalarType> dtype=c10::nullopt, c10::optional<at::Layout> layout=c10::nullopt, c10::optional<at::Device> device=c10::nullopt, c10::optional<bool> pin_memory=c10::nullopt);
+// The below ops don't get a duplicated C++ implementation.
+// They are backward ops, which make them very unlikely to be called directly
+// by external code (at::native::trace_backward).
+// They get their own declaration for BC purposes however.
```

Comment:
@albanD @wconstab wondering if anyone has thoughts on this. Keeping these extra ops for BC to guarantee no internal failures felt like a good idea. But instead of duplicating all of the code, I just routed them to the symint variants (cuts down on code size a bunch but is marginally slower - shouldn't really matter in practice though, since hopefully nobody is calling these).